dist/docs_data.py

```diff
@@ -70,7 +67,7 @@ def __init__(self, doxygen_name, data_structures, files):
     ArchDocPage('arch-python',
         [],
         ['lang/python/']),
-    ArchDocPage('arch-row',
+    ArchDocPage('arch-column-row',
```

Comment:
I think the sort order is wrong here.

Also, on the index page, every title has `(Architecture Guide)` which doesn't seem necessary.

Reply:
Yes good finding, I will re order.
I think it's not too bad to have the prefix "arch" to know where it belongs to. We have quite a few pages as part of the docs.